pallets/phala/src/registry.rs

```diff
@@ -41,6 +42,10 @@ pub mod pallet {
 
 		type UnixTime: UnixTime;
 		type AttestationValidator: AttestationValidator;
+
+		/// Verify attestation, SHOULD NOT SET FALSE ON PRODUCTION !!!
+		#[pallet::constant]
+		type VerifyPRuntime: Get<bool>;
```

Comment:
E2E test is broken on dev test. We need to either change E2E test to register the pruntime whitelist, or put it into the genesis config to disable it on `--dev` mode.

Reply:
verify genesis config is another constant, `VerifyPRuntime` is only verify PRuntime, and will not triggered by `e2e`